core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -116,11 +116,11 @@ object ConsumerGroupCommand extends Logging {
 
```

Comment:
@hachikuji @guozhangwang I noticed that this class was only used in the deprecated admin client, do we still need to add `group.instance.id` support to it? If not, I could consider removing the follow-up changes in scala code.

Reply:
I think @ijuma already have a PR for removing the deprecated admin clients. We can wait for his PR to be merged first?

Reply:
I merged it.